addon-test-support/ember-qunit/index.js

```diff
@@ -24,7 +25,11 @@ import {
   setupApplicationContext,
   teardownApplicationContext,
   validateErrorHandler,
+  getSettledState,
 } from '@ember/test-helpers';
+import { detectPendingTimers, reportPendingTimers } from './async-timer-leak-detection';
+
+const TESTS_WITH_LEAKY_ASYNC = [];
```

Comment:
we should move this into the util file, no need to track it here and pass it around

Reply:
Good point. Not sure why I didn't see this....